modules/internal/ChapelArray.chpl

```diff
@@ -1883,9 +1883,8 @@ module ChapelArray {
      as argument ``that`` and all elements of this array are
      equal to the corresponding element in ``that``. Otherwise
      return false. */
+  @unstable "the 'Array.equals()' method is unstable"
   proc _array.equals(that: _array): bool {
```

Comment:
Note that this warning is now in the spec - should we "no doc" this method instead of leaving an unstable warning in the spec?